src/assets/css/style.css

```diff
@@ -987,4 +987,323 @@ lets go button uses (notified-btn-txt)
   font-family: Poppins-bold;
   font-size: 16px;
   font-weight: 400;
-}
\ No newline at end of file
+}
+
+/* ..............START OF COLOR SETTINGS PAGE */
+/* ..............START OF COLOR MODE BODY */
+.color-mode-display{
```

Comment:
We can't see anywhere it indicates if its a light or dark theme

If this does not need to specify light and dark theme, then lets know